### PR TITLE
Don't link to specific versions of rust in the book.

### PR DESCRIPTION
**Pull request by Stebalien**
They're forever doomed to be out-of-date. Also, don't assume the user is
installing a particular version.

r? @steveklabnik

### CODE REVIEW

File: src/doc/trpl/installing-rust.md [hunk 1]
```diff
@@ -75,10 +72,11 @@ If you've got Rust installed, you can open up a shell, and type this:
 $ rustc --version
 ```
 
-You should see the version number, commit hash, commit date and build date:
+You should see the version number, commit hash, and commit date. If you just
+installed version 1.0.0, you should see:
 
 ```bash
-rustc 1.0.0 (a59de37e9 2015-05-13) (built 2015-05-14)
+rustc 1.0.0 (a59de37e9 2015-05-13)
```
Comment (Stebalien) [inline src/doc/trpl/installing-rust.md]: OK, technically this isn't correct for 1.0 but it is for 1.1+.

**Comment by brson** [comment]
cherry-picking this to 1.1 because the bad instructions bother me

**Comment by brson** [comment]
Actually, nevermind. It doesn't apply cleanly.

**Comment by brson** [comment]
@steveklabnik is going to do the cherry-pick

**Comment by steveklabnik** [comment]
https://github.com/rust-lang/rust/pull/26440

**Pull request merged**